SpatialGDK/Source/SpatialGDK/Public/Interop/ActorSystem.h

```diff
@@ -191,6 +206,8 @@ class ActorSystem
 
 	FChannelsToUpdatePosition ChannelsToUpdatePosition;
 
+	TArray<FObjectRepNotifies> RepNotifiesToSend;
```

Comment:
I'd like it if we considered passing this into methods that need it, as opposed to making it a field of the ActorSystem.

Reply:
+1 if possible

Reply:
Wanna push back a bit on this - this change would mean it is passed to these methods in actorsystem: ProcessAdds, EntityAdded, RefreshEntity, ReceiveActor, ApplyFullState, ApplyComponentDataOnActorCreation, ApplyComponentData, ProcessUpdates, ComponentUpdate, ApplyComponentUpdate, ComponentAdded, HandleIndividualAddComponent, AttachDynamicSubobject, InvokeRepNotifies, ResolvePendingOperations, ResolveIncomingOperations

seems like a tradeoff I guess, but is it worth adding it to all these methods (maybe some could be removed from that list by creating separate lists in receiveactor, refreshentity, processupdates, but don't think it changes much.) Interested in how you approach this tradeoff

Reply:
looking at this also made me notice we sometimes call ResolvePendingOperations originating from tickflush that in turn seems to be called from native. Slightly surprised about this flow, what is its purpose? Do the calls in ApplyFullState/AttachDynamicSubobject not address all cases?

Reply:
If it can't be pulled out simply then it's fine. Generally we want to avoid global transient state but sometimes it's much simpler.